doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -366,6 +366,7 @@ You must install a number of software dependencies to create a suitable build en
 - [Microsoft Visual Studio 2017]( https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=15), which is the default compiler level used by OpenJDK11; or [Microsoft Visual Studio 2013]( https://go.microsoft.com/fwlink/?LinkId=532495), which is chosen to compile if VS2017 isn't installed.
 - [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download)
 - [LLVM/Clang](http://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe)
+- [NASM Assembler](https://www.nasm.us/), which would be installed as one of the packages in the `Devel` category in Cygwin.
```

Comment:
Is there any reason why the Cygwin is picked instead of the official release from https://www.nasm.us/ ?

Reply:
Since the instructions ask to install **all** packages in the Devel category, and the NASM version that would be installed as part of the packages is capable of building it, I thought that would be enough. However, to give us an easier time upgrading to NASM v2.14 in the near future, I have updated the instructions to download from the official website.